### ISSUE TITLE
version 1.0

### ISSUE THREAD

**Issue by gilbertohasnofb** [KoreTeknology/Quadraphonic-Plugins-for-VCV-Rack#2]
Your plugin is not compatible any longer with the current version of VCV Rack. Are you planning on updating it and releasing a new version?

**Comment by KoreTeknology** [comment]
hi Gilbert, yes, i am planning to do a new version, but i havent the time right now :) 
thanks for your interest... i would like to know how you are using it within the 0.6.2 release of vcv ? if you dont mind...

**Comment by gilbertohasnofb** [comment]
Great, thanks for taking a look at it!

> thanks for your interest... i would like to know how you are using it 

In the past I used to write electronic pieces using Pure Data mainly for 4- and 8-channel systems. It's been some time since I last did any multichannel serious work, but this is one of those 'nice-to-have' set of modules, particularly the quad output. Of course I could simply use two regular 2-channel outputs and route them through Jack, but it makes the patch clearer to have it as a single one. Also, I am considering to use Rack for a class of computer music and this might come in handy if any student wants to try using more than 2 channels.

**Comment by gilbertohasnofb** [comment]
Ah, and of course the quad mixer makes it much easier to pan things around the quad space.

**Comment by KoreTeknology** [comment]
yes, you are right... i m also planning to finish my 16 track sequencer to be able to control the matrix, using automations patterns, what do you think about it ?
i am also using VCv to teach people ;) great to ear that you are doing the same

**Comment by gilbertohasnofb** [comment]
That 16-track sequencer with quad output sounds like a great addition to VCV, I'd definitely be interested in it. Thanks again for taking a look at this and good luck with your classes too!

**Comment by pgatt** [comment]
There has been a user going on the VCV community forum (netboy3 is his username) who has in the past been happy to help in the migration of plugins to v1.0 compatibility in VCV. Would you like a hand getting your plugin updated to the current version if he is keen and able?
